etl-pipeline/processes/record_pipeline.py

```diff
@@ -18,13 +19,16 @@ class RecordPipelineProcess:
     def __init__(self, *args):
         self.db_manager = DBManager()
 
-        self.record_queue = os.environ['RECORD_PIPELINE_QUEUE']
-        self.record_route = os.environ['RECORD_PIPELINE_ROUTING_KEY']
+        self.record_queue = os.environ.get('RECORD_PIPELINE_QUEUE')
+        self.record_route = os.environ.get('RECORD_PIPELINE_ROUTING_KEY')
```

Comment:
curious why the move to `get`: I assume `create_or_connect_queue` will still fail if the env var is missing?

Reply:
ya it's a good question - the queue methods will fail. so i guess should we err out in the constructor or later on?

Reply:
I think just getting the key error here would be the most clear, instead of having to decipher why connecting to the queue failed.

Reply:
copy!